doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -1,6 +1,18 @@
 Install OpenCV-Python in Ubuntu {#tutorial_py_setup_in_ubuntu}
 ===============================
 
+
+
+# Table of Contents
+
+1. [Installing OpenCV using pip in Ubuntu](#pip)
+2. [Installing OpenCV-Python from Pre-built Binaries](#binaries)
+3. [Building OpenCV from source](#source)
+4. [Required Built-in Dependencies](#build)
+5. [Optional Dependencies](#dependencies)
+6. [Downloading OpenCV](#download)
+7. [Configuring and Installing](#install)
```

Comment:
Usage is incorrect. Remove completely and keep only `@tableofcontents` - nothing else